Samples.xcodeproj/project.pbxproj

```diff
@@ -1145,6 +1156,7 @@
 				0074ABC428174F430037244A /* Sample.swift in Sources */,
 				E004A6F0284E4B9B002A1FE6 /* DownloadVectorTilesToLocalCacheView.swift in Sources */,
 				4D2ADC4329C26D05003B367F /* AddDynamicEntityLayerView.swift in Sources */,
+				D7EAF35A2A1C023800D822C4 /* SetMinAndMaxScaleView.swift in Sources */,
```

Comment:
Don't forget to add the source code to `Copy Source Code Files` in the Build Phases. You can test if the README is correctly added by tapping on the info button in the sample.

<img width="677" alt="image" src="https://github.com/Esri/arcgis-maps-sdk-swift-samples/assets/9660181/0d702851-ebf6-4db5-abd7-156c347a1d82">